crypto/asn1/x_int64.c

```diff
@@ -77,6 +77,15 @@ static int uint64_c2i(ASN1_VALUE **pval, const unsigned char *cont, int len,
     char *cp;
     int neg = 0;
 
+    /*
+     * Strictly speaking, zero length is malformed.  However, long_c2i
+     * (x_long.c) encodes 0 as a zero length INTEGER (wrongly, of course),
+     * so for the sake of backward compatibility, we still decode zero
+     * length INTEGERs as the number zero.
+     */
+    if (len == 0)
+        goto long_compat;
```

Comment:
No. You are doing a goto that uses cp before cp has been initialised. That needs to be fixed.

Reply:
Ah ok, thanks.  Fixing